docs/contribute/saros-server.md

```diff
@@ -31,6 +31,8 @@ The server will need some kind of permission system to allow a remote user to ma
 
 One server currently represents only one session. To lift this short-coming a Super-Server could be developed, that starts a new session-server on demand. This would also play nicely with the idea of an admin-user, as the user requesting the session could automatically be configured to be the first admin.
 
+This concept could prove to be very memory inefficient, as a new process and JVM would be started for each session. The overhead this brings needs to be evaluated to decide, how this should exactly be implemented. If the additional memory requirements are not overshadowed by Saros overall memory usage, an in-process solution could prove to be more reasonable.
+
```

Comment:
@srossbach I hope this explanation is satisfactory, as it clearly states, that the nature of the actual implementation still remains to be discussed/decided/evaluated.